### PR TITLE
Change FileSet to GenericWork

### LINKED ISSUE #1461: Regression: Batch edit functionality
Compare https://github.com/projecthydra/sufia/blob/6.x-stable/app/views/my/_sort_and_per_page.html.erb with https://github.com/projecthydra/sufia/blob/master/app/views/my/_sort_and_per_page.html.erb

"Edit Selected" and `batch_delete` are gone now.  We want this functionality to work like it did in 6.0

### PR DESCRIPTION
**Pull request by njaffer**
closes #1461 and #1496

### CODE REVIEW

**Comment by mjgiarlo** [comment]
Three batch-edit-related failures in the test suite.

**Comment by njaffer** [comment]
@mjgiarlo, yeah forgot about the spec tests! but now it should be all good

File: app/views/batch_edits/edit.html.erb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <%= render_breadcrumbs builder: Sufia::BootstrapBreadcrumbsBuilder %>
-<h2 class="non lower">Batch Edit Descriptions &nbsp;&nbsp;&nbsp;<small>Click on labels below to edit file descriptions.</small> </h2>
+<h2 class="non lower">Batch Edit Descriptions &nbsp;&nbsp;&nbsp;<small>Click on labels below to edit work descriptions.</small> </h2>
+
 <div class="scrollx scrolly fileHeight"> <!-- original values -->
   <h3> <b>Changes will be applied to: (<%= @names.size %> files) </b></h3>
```
Comment (mjgiarlo) [inline app/views/batch_edits/edit.html.erb]: @njaffer There's a reference to files here as well.

Comment (njaffer) [inline app/views/batch_edits/edit.html.erb]: ahh ... good catch!

**Comment by njaffer** [comment]
@mjgiarlo , @jcoyne: Can this be merged ?

**Comment by mjgiarlo** [comment]
I'm :+1: on it. @jcoyne, any comments/questions?

**Pull request merged**